### PR TITLE
Piyush_Assignment_8_Done

### PR DESCRIPTION
**Pull request by PiyushPJun20**
Piyush_Assignment_8_Done

### CODE REVIEW

File: src/BankingSystem.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	static int userDebtCount;
+	static int userprintCount;
+
+	void debitAccount(int debitAmt){
+        debitCount++;
+		userDebtCount++;
+	}
+	void creditAccount(int creditAmt){
+		creditCount++;
+		userCrdtCount++;
```
Comment (KrishnaKTechnocredits) [inline src/BankingSystem.java:27]: You forgot to perform an operation in debitAcc() and credit() METHODS. You should have a balance instance variable. You should write the condition to verify the insufficient balance.

File: src/BankingSystem.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	}
+	void printBalance(int crAmt,int debtAmt){
+	    System.out.println(""); 
+	    System.out.println(" Current Balance is : " + ((crAmt-debtAmt)));
```
Comment (KrishnaKTechnocredits) [inline src/BankingSystem.java]: This line not required here.

Comment (PiyushPJun20) [reply src/BankingSystem.java]: Changes Incorporated.

File: src/BankingSystem.java (hunk 1 above)
Comment (PiyushPJun20) [reply src/BankingSystem.java:27]: Changes Incorporated.

**Review by KrishnaKTechnocredits — COMMENTED** [review]
Code looks good to me.

**Pull request merged**